### PR TITLE
[Bug 671216] Added avatar next to the username in the PM

### PR DESCRIPTION
**Pull request by anushbmx**
This adds user avatar next to username in the PM auto-complete box.

### CODE REVIEW

**Comment by mythmon** [comment]
LGTM. Thanks!

File: kitsune/users/api.py [hunk 1]
```diff
@@ -65,7 +65,8 @@ def usernames(request):
             users = users.filter(last_login__gte=last_login)
 
         return [{'username': u.username,
-                 'display_name': display_name_or_none(u)}
+                 'display_name': display_name_or_none(u),
+                 'avatar':profile_avatar(u,24)}
```
Comment (rehandalal) [inline kitsune/users/api.py]: nit: space here

Comment (rehandalal) [inline kitsune/users/api.py]: `'avatar': profile_avatar(u, 24)}`

**Comment by anushbmx** [comment]
@rehandalal i added the space now the test failed .?

**Pull request merged**